backend/typescript/models/waitlistedCamper.model.ts

```diff
@@ -0,0 +1,45 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+export interface WaitlistedCamper extends Document {
+  id: string;
+  firstName: string;
+  lastName: string;
+  age: number;
+  parentName: string;
+  contactEmail: string;
+  contactNumber: string;
+  camps: Schema.Types.ObjectId;
```

Comment:
```suggestion
  camp: Schema.Types.ObjectId;
```
only storing a single camp here since each waitlist registration is for one camp!